students/alex_skrn/lesson09/mailroom.py

```diff
@@ -0,0 +1,367 @@
+#!/usr/bin/env python3
+
+"""Mailroom - Lesson 9 - Object Orientation."""
+import os
+import datetime
+import tkinter as tk
+from tkinter import filedialog
+
+data = {'Aristarkh Lentulov': [4.5, 5.0],
+        'El Lissitzky': [34.2, 30.0, 35.5],
+        'Kazimir Malevich': [15.0, 20.25, 12.25],
+        'Marc Chagall': [148.75, 155.0],
+        'Wassily Kandinsky': [75.0, 50.5, 60.4],
+        }
+
+
+####################
+# SINGLE DONOR CLASS
+####################
+class SingleDonor(object):
+    """Provide a class for a single donor."""
+
+    def __init__(self, name, donation):
+        """Instantiate a SingleDonor class object."""
+        self._name = name
+        if isinstance(donation, list):
+            self._donations = donation
+        elif isinstance(donation, tuple):
+            self._donations = list(donation)
+        else:
+            self._donations = [donation]
+
+    @property
+    def name(self):
+        """Provide a getter method for the name property."""
+        return self._name
+
+    @property
+    def donations(self):
+        """Provide a getter method for the donations property."""
+        return self._donations
+
+    @staticmethod
+    def sort_by_total(self):
+        """Provide a sort_key for sorting by total donations."""
+        return sum(self._donations)
+
+    @staticmethod
+    def sort_by_name(self):
+        """Provide a sort_key for sorting by name."""
+        return self._name
```

Comment:
same here.  `staticmethods` are methods that don't refer to the object at all, and therefore you don't need to instantiate the class beforehand to use them.